src/Cake.Testing/FakeEnvironment.cs

```diff
@@ -43,6 +43,15 @@ public sealed class FakeEnvironment : ICakeEnvironment
         /// <value>The runtime Cake is running in.</value>
         public FakeRuntime Runtime { get; }
 
+        /// <inheritdoc/>
+        ILaunchInfo ICakeEnvironment.LaunchInfo => LaunchInfo;
```

Comment:
Is there a technical reason you're having to implement this twice? [See comment](https://github.com/cake-build/cake/pull/3132/files#r587071298)

Reply:
I followed the same pattern as `Runtime` and `Platform`